---
title: "Classifying DNA mismatch repair variant pathogenicity using protein structure and function predictions"
author: "Assoc. Prof. Bernard Pope"
summary: "The DNA mismatch repair (MMR) mechanism plays an important role in maintaining genome integrity by correcting small mutations that can arise from DNA damage or from mistakes made during replication and recombination"
date: 2019-03-27T16:22:26+11:00
draft: true
tags: [
    "VariantsPrediction",
    "tools",
]
---

Assoc. Prof. Bernard Pope\
Melbourne Bioinformatics, The University of Melbourne

## Abstract

The DNA mismatch repair (MMR) mechanism plays an important role in maintaining genome integrity by correcting small mutations that can arise from DNA damage or from mistakes made during replication and recombination. Loss of MMR function in a cell can lead to a rapid accumulation of somatic mutations, giving rise to a hypermutated phenotype. In humans this mutational process can lead to cancer when key oncogenes are disrupted. Individuals who carry a pathogenic germline mutation in an MMR gene are at a substantially higher lifetime risk of many types of cancer, a condition known as Lynch syndrome. Therefore, considerable effort has been made over many years in identifying pathogenic MMR gene variants in the population.

A major challenge for this work has been the discrimination of high-risk deleterious variants from those which are benign. Addressing this challenge, the International Society for Gastrointestinal Hereditary Tumours (InSiGHT) has established a database of expertly curated and classified MMR variants (www.insight-database.org). The database contains a collection of over 2600 unique MMR gene variants associated with pathogenicity classifications. However, nearly 800 of those variants fall into the "uncertain" class, posing challenges for variant interpretation.

In this talk I will describe our efforts to incorporate predicted protein structural and functional features of MMR variants to develop a new computational classifier of pathogenicity. I will also demonstrate our new website, called StruMMR (biosig.unimelb.edu.au/strummr), for visualising and interpreting the results of the classifier. We believe this information will provide further guidance to the interpretation of novel variants not currently classified within the InSiGHT database and assist with re-classification of variants within the "uncertain" class.
