---
title: "Mitch: Multivariate pathway enrichment for multi-omics and single-cell profiling data"
author: "Mark Ziemann"
summary: "Inference of biological pathway activity via gene set enrichment analysis is a frequently used approach in the interpretation of clinical and other omics data"
date: 2019-06-12T16:56:39+11:00
draft: true
tags: [
    "R",
    "tools",
    "GeneSetEnrichment",
]
---

Mark Ziemann\
Deakin University, Geelong, School of Life and Social Sciences

## Abstract

Inference of biological pathway activity via gene set enrichment analysis is a frequently used approach in the interpretation of clinical and other omics data. With the proliferation of new profiling approaches and ever-growing size of data sets generated, there is greater need for researchers to distill these data into useful findings. We developed an R package for identification of pathway enrichment in multi-omics data called Mitch. Mitch uses a rank-MANOVA based statistical approach to detect sets of genes that exhibit enrichment in the multidimensional space as compared to the background. The package provides visualisation features that enable exploration of enrichments in two or more dimensions. We describe four applications where Mitch is useful. In the first case the effect of drug in reversing a disease associated gene expression signature. Secondly, an integrative analysis of transcriptome and epigenome profiling data to identify transcription factors associated with gene regulation in response to interferon treatment. Thirdly, a comparison of pathway level regulation of the same RNA-seq data seq after differential expression analysis with three commonly used tools. Lastly, the identification of coordinated pathway-level expression changes in populations of individual cells in a mouse model of disease

